### ISSUE TITLE
make  -a as default ?

### ISSUE THREAD

**Issue by avichou** [buo/brew-cask-upgrade#232]
is it possible to make brew CU update all casks (included those with autod update on) by default ? 

so that I won't have to type -a each time, and will simply have to enter brew cu -y 

(side question , is it dangerous/not advised to update apps that are marked as "auto update" ? )

**Comment by yurikoles** [comment]
Persistent config is being implemented in #174

**Comment by ondrejfuhrer** [comment]
> (side question , is it dangerous/not advised to update apps that are marked as "auto update" ? )

Not really, it's up to every person to decide, how they want to manage those casks, either through `brew` or let them auto-update. You can also combine both of the approaches, etc.

**Comment by vasil-nesterov** [comment]
As a new user who just found this hidden gem, I'd say the default behavior is non-intuitive.

This morning, Notion and Slack asked me for a password to auto-update once again, and I searched for a solution.
I wanted a better way to auto-update all the annoying auto-updating apps, and initially I was disappointed that this tool wasn't doing the job for me. I came here to raise or find an existing issue, and landed in this discussion.

Now I'm all set, but maybe there are many other people who don't RTFM and share the same experience.
